wandb/sdk/wandb_init.py

```diff
@@ -270,7 +270,9 @@ def _jupyter_teardown(self):
         ipython = self.notebook.shell
         self.notebook.save_history()
         if self.notebook.save_ipynb():
-            self.run.log_code(root=None)
+            art = self.run.log_code(root=None)
```

Comment:
you need to only do a "cleanup" if run.log_code returns non None
(why isnt mypy complaining about this.. i think annotations might be broken)

Reply:
i know the answer.  wandb_init.py annotations are coming tomorrow (they are in the grpc branch)

Reply:
actually, wandb_init.py isnt typed in my branch :(

Reply:
Ah, my mistake. Fixed in my branch, I'll try to put up a typing PR this week